SchoolBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace School
+{
+    public abstract class SchoolBase : Sorting
```

Comment:
What is SchoolBase used for? And why on earth does it derive from Sorting?

I'd say it should be merged with Shool and use a Sorting class inside. (which should be made static).

Reply:
ok